sdk/iot/core/inc/az_iot_core.h

```diff
@@ -21,12 +21,16 @@
 /**
  * @brief Azure IoT service MQTT connection properties.
  *
+ * @note You MUST NOT use any symbols (macros, functions, structures, enums, etc.)
+ * prefixed with an underscore ('_') directly in your application code. These symbols
+ * are part of Azure SDK's internal implementation; we do not document these symbols
+ * and they are subject to change in future versions of the SDK which would break your code.
  */
+
```

Comment:
Remove new line?

Reply:
Moved the entire comment in the file section.